app/src/main/kotlin/com/wire/android/ui/common/UserProfileAvatar.kt

```diff
@@ -44,26 +45,14 @@ fun UserProfileAvatar(
             contentDescription = stringResource(R.string.content_description_user_avatar),
             contentScale = ContentScale.Crop,
             modifier = Modifier
+                .padding(dimensions().userAvatarStatusBorderSize)
                 .background(Color.Black, CircleShape)
                 .size(size)
         )
-        Box(
-            contentAlignment = Alignment.Center,
+        UserStatusDot(
```

Comment:
Dot? Can we try to rename to something that is not design specific so that we avoid possible refactorings if designs change in the future? 
Maybe `UserStatusIndicator`, `UserStatusBadge` or `UserStatusMarker` ?

Reply:
`Indicator` sounds good to me